libsurfman/src/surface.c

```diff
@@ -135,6 +155,58 @@ static void update_mapping (struct surface_priv *p, size_t len)
   p->len = len;
 }
 
+static int compar_pfns(const void *a, const void *b)
```

Comment:
I see that `qsort`, which uses this, calls its function parameter `compar` with the `e` missing, and then in the man page, the example function is called `cmp<somthing>` at which point those people just lost the high ground on naming.
Calling this function `compare_pfns` would be slightly nicer for non-unix-speaking humans.